src/System.Drawing.Common/tests/Helpers.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.IO;
+using System.Reflection;
+
+namespace System.Drawing.Tests
+{
+    public static class Helpers
+    {
+        public static string GetEmbeddedResourcePath(string name)
+        {
+            return Path.Combine(Assembly.GetExecutingAssembly().Location, "..", "bitmaps", name);
```

Comment:
Is there a significance to these being embedded resources if they are just copied to the output directory and used as files?